google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/FirestoreImpl.java

```diff
@@ -248,6 +248,99 @@ public void onCompleted() {
     return futureList;
   }
 
+  void getAll(
```

Comment:
Please merge this with the internal method above. You can likely rewrite the existing `getAll()` to use your new method.